packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/lib/simple.dart

```diff
@@ -35,7 +35,13 @@ class Model {
   final String value;
 }
 
-@JsonSerializable()
+enum TestEnum {
+  one,
+  two,
+  three;
+}
+
+@JsonSerializable(createPerFieldToJson: true)
```

Comment:
Could you remove the flag and update the `build.yaml` instead?

Reply:
Not until we fix the issue with getting the perFieldToJson value